### PR TITLE
hotokami.jp

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Add your comment and screenshots

```
!https://hotokami.jp/
!PC
!ad
hotokami.jp##.shitennoji-big-banner
hotokami.jp##div[class*="adsense"]
hotokami.jp##div[class="top_recent_article"][style="display:block;"]
hotokami.jp##.layout_fix_advice--apps
!annoyance
hotokami.jp##.mail_magazine_form--container
!
!https://hotokami.jp/area/tokyo/Hkrta/Hkrtaty/Dmayy/80478/
!PC
!ad
hotokami.jp##div[class^="full_screen_ad"]
!social
hotokami.jp##.spots__share
!
!https://hotokami.jp/articles/169/
!ad
hotokami.jp##.shitennoji-banner-js
!social
hotokami.jp##.article-sns-box__wrapper
```

https://hotokami.jp/
<details><summary>screenshot</summary>

<a href="https://ibb.co/q7sxqXm"></a>
<a href="https://ibb.co/CJK4ss0"></a>
<a href="https://ibb.co/12ZGftH"></a>
<a href="https://ibb.co/1vNktrR"></a>
</details><br/>

 https://hotokami.jp/area/tokyo/
<details><summary>screenshot</summary>

<a href="https://ibb.co/LCWR4Jp"></a>
</details><br/>

https://hotokami.jp/area/tokyo/Hkrta/Hkrtaty/Dmayy/80478/
<details><summary>screenshot</summary>

<a href="https://ibb.co/D42RNZn"></a>
<a href="https://ibb.co/K5k7GxV"></a>
</details><br/>

https://hotokami.jp/articles/169/
<details><summary>screenshot</summary>

<a href="https://ibb.co/kGbdTTs"></a>
<a href="https://ibb.co/QD80qxF"></a>
</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -40,6 +40,12 @@ yutura.net,goetheweb.jp,mono-money.com,tetsudo-ch.com,home.kingsoft.jp,coco-tte.
 !
 !
 seijiyama.jp###ad-two-center
+hotokami.jp##.shitennoji-banner-js
+hotokami.jp##.shitennoji-big-banner
+hotokami.jp##.layout_fix_advice--apps
```
Comment (zloyden) [inline JapaneseFilter/sections/specific.txt:45]: Please move this to AdGuard Annoyances - self-promo.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt]: ok,done

**Review by zloyden — APPROVED** [review]

**Comment by Yuki2718** [comment]
Just hiding `.full_screen_ad` causes unresponsive tabs after you clicked a few times.
Relevant code:
```
document.addEventListener("turbo:load", function() {
    if (sessionStorage.getItem("accessTimes")) {
        sessionStorage.setItem('accessTimes', parseInt(sessionStorage.getItem("accessTimes")) + 1);
        if (parseInt(sessionStorage.getItem("accessTimes")) % 3 == 0) {
            document.querySelectorAll("a").forEach(function(link) {
                if (!link.hasAttribute("data-turbo")&&!link.href.includes("#")&&!link.href.includes(".jpg")&&!link.hasAttribute("data-next")&&!link.href.includes("features")&&!link.href.includes("users")) {
                    link.dataset.href = link.href
                    link.classList.add("pointer-link")
                    link.removeAttribute("href");
                    link.addEventListener("click", function() {
                        if (document.querySelector(".full_screen_ad") != null) {
                            document.querySelector(".full_screen_ad").style.display = "flex";
                            document.querySelector(".full_screen_ad__close").href = link.dataset.href;
                        } else {
                            //.full_screen_adが取得できなかった場合
                            link.href = link.dataset.href
                        }
                    })
                }
            })
        }
    } else {
        sessionStorage.setItem('accessTimes', 1);
    }
}, false);
```
You can use [remove](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#pseudo-class-remove-and-pseudo-property-remove) in this case, as they give up displaying ad and let the tabs work normally if they can't find `.full_screen_ad`. I'll fix.
(BTW `.full_screen_ad` is the top parent and all other `div[class^="full_screen_ad"]` are descendants, meaning `##.full_screen_ad` is enough.)

**Comment by SKEIDs** [comment]
@Yuki2718 
thank you for fix of my rule.

**Pull request merged**